lldb/source/Plugins/Language/CPlusPlus/GenericBitset.cpp

```diff
@@ -91,7 +91,7 @@ lldb::ChildCacheState GenericBitsetFrontEnd::Update() {
   size_t size = 0;
 
   if (auto arg = m_backend.GetCompilerType().GetIntegralTemplateArgument(0))
-    size = arg->value.getLimitedValue();
+    size = arg->value.getInt().getLimitedValue();
```

Comment:
Technically I think these need to check that the value is an integral type as a malicious could crash lldb by declaring a `std::bitset` with a floating point template argument.